vms/platformvm/txs/fee/complexity.go

```diff
@@ -103,42 +112,32 @@ var (
 			wrappers.IntLen + // genesis length
 			wrappers.IntLen + // subnetAuth typeID
 			wrappers.IntLen, // subnetAuthCredential typeID
-		gas.DBRead:  1,
-		gas.DBWrite: 1,
-		gas.Compute: 0,
+		gas.DBRead:  3, // get subnet auth + check for subnet transformation + check for subnet conversion
+		gas.DBWrite: 1, // put chain
 	}
 	IntrinsicCreateSubnetTxComplexities = gas.Dimensions{
 		gas.Bandwidth: IntrinsicBaseTxComplexities[gas.Bandwidth] +
 			wrappers.IntLen, // owner typeID
-		gas.DBRead:  0,
-		gas.DBWrite: 1,
-		gas.Compute: 0,
+		gas.DBWrite: 1, // write subnet owner
 	}
 	IntrinsicImportTxComplexities = gas.Dimensions{
 		gas.Bandwidth: IntrinsicBaseTxComplexities[gas.Bandwidth] +
 			ids.IDLen + // source chainID
 			wrappers.IntLen, // num importing inputs
-		gas.DBRead:  0,
```

Comment:
what about the imported inputs?

Reply:
is that handled as part of I/O complexity?

Reply:
These are added here: https://github.com/ava-labs/avalanchego/blob/acp-103-finalize-complexities/vms/platformvm/txs/fee/complexity.go#L588